protocol/hid/fan.c

```diff
@@ -146,14 +146,16 @@ int corsairlink_hid_fan_speed(struct corsair_device_info *dev, struct libusb_dev
 	commands[++ii] = CommandId++;
 	commands[++ii] = ReadTwoBytes;
 	commands[++ii] = FAN_ReadRPM;
+	commands[++ii] = 0x00;
+	commands[++ii] = 0x00;
 
 	commands[0] = ii;
 
 	rr = dev->driver->write(handle, dev->write_endpoint, commands, ii);
-	rr = dev->driver->read(handle, dev->read_endpoint, response, 32);
+	rr = dev->driver->read(handle, dev->read_endpoint, response, 64);
```

Comment:
Changing the bytes read from the USB buffer shouldn't change the first 6 bytes of the response.